client/src/components/PostedOn.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Typography } from "@mui/material";
+import { format } from "date-fns";
+import { useTheme } from "@emotion/react";
+
+//assuming the format of the props.date is YYYY-MM-DD
+const PostedOn = (props) => {
+  const postedDate = format(new Date(props.date), "M/d/yy");
+  let theme = useTheme();
+  return (
+    <p
```

Comment:
For the sake of consistency, i think the mui `Typography` component with the variant for `body` (or whatever appropriate variant) would help cover any base styling we have set at the theme level as well.

https://mui.com/material-ui/api/typography/